jupyter_conda/envmanager.py

```diff
@@ -524,6 +524,29 @@ async def env_packages(self, env: str) -> Dict[str, List[str]]:
 
         return {"packages": [normalize_pkg_info(package) for package in data]}
 
+    async def pkg_depends(self, pkg: str) -> Dict[str, List[str]]:
+        """List environment packages dependencies.
+
+        Args:
+            pkg (str): Package name
+
+        Returns:
+            {"package": List[dependencies]}
+        """
+        resp = {}
+        ans = await self._execute(self.manager, "repoquery", "depends", "--json", pkg)
```

Comment:
As conda does not support this, the manager should be tested. And in this particular case a default value like {_name_: `null`} to signify it can't be done is a good candidate.